clients/vault/src/oracle/storage/impls.rs

```diff
@@ -230,6 +230,82 @@ impl ScpArchiveStorage {
 	}
 }
 
+impl TransactionsArchiveStorage {
+	pub async fn get_transactions_archive(slot_index: i32) -> Result<XdrArchive<TransactionHistoryEntry>, Error> {
+		let (url, file_name) = Self::get_url_and_file_name(slot_index);
+		//try to find xdr.gz file and decode. if error then download archive from horizon archive
+		// node and save
+		println!("{url}");
+		println!("{file_name}");
```

Comment:
Consider either removing these log statements or unifying them with more context (ie making them more descriptive) into a single line. Just printing the variables without context might be confusing when running the vault client.